cni/scripts/cleanupnetwork.ps1

```diff
@@ -0,0 +1,76 @@
+$Global:ClusterConfiguration = ConvertFrom-Json ((Get-Content "c:\k\kubeclusterconfig.json" -ErrorAction Stop) | out-string)
```

Comment:
this works only for k8s cluster.. can we make this script work irrespective of orchestrator?